sql/persistent_page_ids.sql

```diff
@@ -0,0 +1,7 @@
+
+CREATE TABLE IF NOT EXISTS /*_*/persistent_page_ids (
+	page_id        INTEGER         NOT NULL,
```

Comment:
MW uses `INT(10) UNSIGNED` for `page_id`, but this is not supported in SQLite. `INTEGER` ends up being `INT(11)` in MySQL.

This lets the table schema work on MySQL/MariaDB, and SQLite which is used in CI. However, we should probably run CI with MySQL as the primary DB. We're not explicitly supporting SQLite, so making sure MySQL works is more important.

Reply:
For extensions with own tables it is nice to run tests against all supported DB types on GH Actions. AFAIK we do not have this set up anywhere any more.